test/util/discovery/discovery.go

```diff
@@ -31,18 +31,23 @@ func DiscoveredResources(clients *client.ClientSet, sriovNodes []string, operato
 	}
 
 	for _, policy := range policyList.Items {
-		if !filter(policy) {
+		if !filterPolicy(policy) {
 			continue
 		}
 		resourceName := policy.Spec.ResourceName
 		for _, node := range nodes {
+			ok, device := filterDevices(node)
```

Comment:
Normally the `ok` value is the second one returned (like in maps, type assertions, etc)